src/utils/jsonTransform.js

```diff
@@ -0,0 +1,64 @@
+import {isAffirmative} from './utils';
+
+export function to_registration_service_format(src) {
+    return {
+        disaster_id: src.basicInfo.disasterIndex, // exists in UI debug, but not in test-data.json TODO
+        // "preferred_language": TODO
+        phone: src.basicInfo.phone,
+        email: src.basicInfo.email,
+        residential_address: src.basicInfo.residenceAddress,
+        mailing_address: src.basicInfo.mailingAddress,
+        county: src.basicInfo.county, // should it be disasterCounty?
```

Comment:
I'm not entirely sure, I thought the guidance was that the user could enter whatever county they resided in, 
regardless of whether or not that county was affected by the disaster. If that isn't the case, we don't need to ask the user for this data, we can use the value they've selected during the `pre-registration` step.

We should make an issue to have this clarified and documented.